media/css/mozorg/home/takeover.less

```diff
@@ -0,0 +1,249 @@
+        padding: 0 (@baseLine * 3);
+        max-width: 960px;
+
+        @media only screen and (max-width: @breakTablet) {
+            padding: 0 10px;
+        }
+    }
+
+    .copy {
+        margin-top: (@baseLine * 8);
+        padding-bottom: (@baseLine * 2);
+
+        @media only screen and (max-width: @breakTablet) {
+            margin-top: (@baseLine * 3);
+        }
+
+        @media only screen and (max-width: 360px) {
+            margin-top: @baseLine;
+        }
+
+        header {
+            width: 540px;
+
+            @media only screen and (max-width: @breakDesktop) {
+                width: 440px;
+            }
+
+            @media only screen and (max-width: @breakTablet) {
+                width: auto;
+            }
+        }
+    }
+
+    form {
+        width: 400px;
+
+        @media only screen and (max-width: @breakTablet) {
+            width: 100%;
+        }
+    }
+
+    .donate-options {
+
+        label {
+            display: inline-block;
+            background-color: @optionInactive;
+            color: @foregroundColor;
+            margin: 5px;
+            padding: 10px;
+            border-radius: 6px;
+            width: 90px;
+            .font-size(1.2rem);
+            text-align: center;
+            cursor: pointer;
+
+            @media only screen and (max-width: @breakTablet) {
+                width: 70px;
+            }
+
+            @media only screen and (max-width: 360px) {
+                margin: 5px 2px;
+                width: 130px;
+            }
+
+            &:hover,
+            &:focus,
+            &.selected {
+                background-color: @optionActive;
+            }
+        }
+
+        input[type="text"] {
+            width: 197px;
+            height: 47px;
+            .font-size(1.2rem);
+
+            @media only screen and (max-width: @breakTablet) {
+                display: block;
+                margin: 10px auto;
+                width: 340px;
+            }
+
+            @media only screen and (max-width: 360px) {
+                width: 270px;
+            }
+        }
+
+        label.own-amount,
+        input[type="radio"] {
+            .visually-hidden;
+        }
+    }
+
+    .recurring {
+        margin-top: 10px;
+
+        label {
+            margin-right: @baseLine;
+            .font-size(1rem);
+        }
+    }
+
+    input[type="submit"] {
+        display: block;
+        background-color: @mozRed;
+        color: @foregroundColor;
+        margin-top: @baseLine;
+        padding: 5px 10px;
+        border: 2px solid @foregroundColor;
+        border-radius: 6px;
+        .font-size(1.5rem);
+        font-weight: 300;
+        text-transform: uppercase;
+        cursor: pointer;
+
+        @media only screen and (max-width: @breakTablet) {
+            margin: @baseLine auto;
+        }
+
+        &:hover,
+        &:focus {
+            background-color: #fff;
+            color: #333;
+            border: 1px solid #333;
```

Comment:
This `1px` is causing the takeover to do a little jump when you hover over submit.